### DISCUSSION TITLE
Missing Authorization header

### DISCUSSION THREAD

**Discussion by ChristopherL-STCU** [microsoft/semantic-kernel#6385] [Q&A] [answered]
I'm trying to get a basic example running. Everything works using Open AI but when I switch to Azure Open AI I receive an error.

> RequestFailedException: Service request failed.
> Status: 400 (Missing Authorization header)
> Content:
> {"status": "Auth token must be passed as a header called Authorization"}

I've deployed a Phi-3-mini-4k-instruct model is Azure AI Studio. I can use the model within AI Studio. And I can create a simple console app using the suggested code within AI Studio. But when I try to use the model with Semantic Kernel I receive the 400 response above.

```
kernelBuilder.AddAzureOpenAIChatCompletion(config["OpenAI:AzureModelId"]!, config["OpenAI:AzureEndpoint"]!, config["OpenAI:AzureApiKey"]!);
```

What am I doing wrong? Is it because I'm using a Phi model?

**Comment by rogerbarreto** [comment, marked as answer]
`AddAzureOpenAIChatCompletion` is intended for only [Azure OpenAI Studio Deployments](https://oai.azure.com/).
Using the [Azure AI Studio](https://ai.azure.com/) will work only for OpenAI connector if the deployed model also supports the same ChatCompletion API as OpenAI, which seems the case for Phi-3. 

Currently we are in works of providing a specific Connector for Model-as-a-Service deployments in Azure AI Studio.

**Comment by jphgardner** [comment]
Any clues when this will be finished?

**Reply by evchaki** [reply]
This was merged for Python last night - https://github.com/microsoft/semantic-kernel/pull/6742

**Reply by jphgardner** [reply]
Ah sorry, Im using C#? @evchaki

**Comment by MD-V** [comment]
I have the same problem with .NET SemanticKernel 1.16.0. Model is hosted serverless with Azure KI Studio.

Error:

Microsoft.SemanticKernel.HttpOperationException: 'Service request failed.
Status: 400 (Missing Authorization header)

Content:
{"status": "Auth token must be passed as a header called Authorization"}
